candidate/models.py

```diff
@@ -1376,6 +1379,11 @@ def create_candidate_row_entry(self, update_values):
                 new_candidate.candidate_twitter_handle = candidate_twitter_handle
                 new_candidate.candidate_url = candidate_url
                 new_candidate.facebook_url = facebook_url
+                new_candidate.photo_url = photo_url
+                if new_candidate.photo_url:
+                    # save_to_candidate_object = False
+                    candidate_results = self.modify_candidate_with_organization_endorsements_image(new_candidate, photo_url,
+                                                                                             False)
                 new_candidate.save()
```

Comment:
I think before saving "new_candidate" we need to extract new_candidate from candidate_results that comes from "modify_candidate_with_organization_endorsements_image", yes?